### PR TITLE
docs: monitoring tools documentation

### PR DESCRIPTION
**Pull request by v-rocheleau**
- Fixed missing instructions in migration guide
- Detailed monitoring tools documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ that make up the Bento platform.
 * [Development](./docs/development.md)
 * [Troubleshooting guide](./docs/troubleshooting.md)
 * [Deployment](./docs/deployment.md)
+* [Monitoring](./docs/monitoring.md)
```
Comment (davidlougheed) [inline README.md:39]: unrelated to this PR, but after this line (or in the section below perhaps, or in a new section called `Customization`) can you add a link to the public discovery config docs?

File: docs/monitoring.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+Grafana is configured to only let in authenticated users from Bento's Keycloak realm, 
+if they have the required client permissions for Grafana.
+
+Create the Grafana OIDC client, its permissions and group mappings with the following:
+
+```bash
+./bentoctl.bash init-auth
+```
+
+Set the outputed value for `BENTO_GRAFANA_CLIENT_SECRET` in the `local.env` file and restart Grafana.
```
Comment (davidlougheed) [inline docs/monitoring.md:38]: outputted*

File: docs/monitoring.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        - Editor permissions
+        - Administration of Grafana
+    - Editor
+        - Viewer permissions
+        - Can explore logs
+        - Can create dashboards for viewers
+    - Viewer
+        - Can view created dashboards
+
+The `admin`, `editor` and `viewer` roles are Grafana concepts. During authentication, Grafana will synchronize the user's roles from Keycloak, and only let the user in if a valid role can be retrieved from the ID token.
```
Comment (davidlougheed) [inline docs/monitoring.md:60]: wrap line to 120chars

**Review by davidlougheed — CHANGES_REQUESTED** [review]
great to have! 2 minor points, otherwise good to go.

File: README.md (hunk 1 above)
Comment (v-rocheleau) [reply README.md:39]: Missed this, good point.

**Review by davidlougheed — APPROVED** [review]
lgtm

**Pull request merged**